res/openvpn-gui-res-ru.rc

```diff
@@ -411,17 +411,17 @@ BEGIN
     IDS_NFO_USAGE "--help\t\t\t: Отобразить это сообщение.\n\
 --connect cnn \t\t: Подключиться к ""cnn"" при запуске. (включая расширение файла)\n\
 \t\t\t   Пример: openvpn-gui --connect office.ovpn\n\
---command cmd [args]\t: Send a command to a running instance of the GUI\n\
+--command cmd [args]\t: Послать команду работающему процессу графического интерфейса\n\
 Supported commands:\n\
-    connect cnn            \t: connect the config named ""cnn""\n\
-    disconnect cnn         \t: disconnect the config named ""cnn""\n\
-    reconnect cnn          \t: reconnect the config named ""cnn""\n\
-    disconnect_all         \t: disconnect all connected configs\n\
-    exit                 \t\t: terminate the running GUI instance (may ask for confirmation)\n\
-    status cnn           \t\t: show the status window of config ""cnn"" if connected\n\
-    silent_connection [0|1]\t: set the silent_connection flag on (1) or off (0)\n\
-    import path          \t\t: Import the config file pointed to by path\n\
-\t\t\tExample: openvpn-gui.exe --command disconnect myconfig\n\
+    connect cnn            \t: подключиться к конфигурации под названием ""cnn""\n\
```

Comment:
может быть "подключиться к " .... "отключиться от .... "
и, кстати, английское тоже можно поменять ))

Reply:
@selvanair do you have an opinion on that? I am not a native English speaker but indeed

`connect cnn \t: connect the config named ""cnn""`

could be changed to 

`connect cnn \t: connect to ""cnn""`

and same for other actions.

Reply:
Current: `connect the config named "cnn"`
Proposed: `connect to "cnn"`

IMO, which is better depends on how configs are named. For example, config files could be named after username or common-name  like "Jane Doe" (I do this) as opposed to the provider name. In which case `connect the config named "foo"` makes a lot more sense than `connect to "foo"`. 

Also, the English version provides info and context to translators, so  some verbosity in there is not a bad thing. Its translators job **not** to do word-for-word translation. Even otherwise, detailed messages are better than terse phrases --- assuming messages are shown only when absolutely necessary and when user is likely to read them. This one is shown in response to --help option.

All that said, I'm okay with the proposed change if that would help anyone.